entrypoint.sh

```diff
@@ -1,12 +1,26 @@
 #!/bin/bash
 
-set -e
+# Function to URL-encode a string
+urlencode() {
+  local string="$1"
+  local encoded=""
+  local length="${#string}"
+  for ((i = 0; i < length; i++)); do
+    local char="${string:i:1}"
+    case "$char" in
+    [a-zA-Z0-9.~_-]) encoded+="$char" ;;
+    *) encoded+="%"$(printf "%02X" "'$char") ;;
+    esac
+  done
+  echo "$encoded"
+}
 
 # This supports both the old environment variables and new ones specific to the Figma setup.
 if [ -n "$PG_FIGMA_FOR_JIRA_DB_ROLE" ]; then
-  DATABASE_URL=postgresql://${PG_FIGMA_FOR_JIRA_DB_ROLE}:${PG_FIGMA_FOR_JIRA_DB_PASSWORD}@${PG_FIGMA_FOR_JIRA_DB_HOST}:${PG_FIGMA_FOR_JIRA_DB_PORT}/${PG_FIGMA_FOR_JIRA_DB_SCHEMA}
+  DATABASE_URL="postgresql://${PG_FIGMA_FOR_JIRA_DB_ROLE}:${PG_FIGMA_FOR_JIRA_DB_PASSWORD}@${PG_FIGMA_FOR_JIRA_DB_HOST}:${PG_FIGMA_FOR_JIRA_DB_PORT}/${PG_FIGMA_FOR_JIRA_DB_SCHEMA}"
```

Comment:
**question:** @elainefigma , do we need to apply the same change here as well to avoid mismatch between configurations? I will verify the behaviour on our side and can apply a fix if required.